test/integration/functional_test.go

```diff
@@ -564,6 +566,63 @@ func validateMySQL(ctx context.Context, t *testing.T, profile string) {
 	}
 }
 
+// validateFileSync start up VM and test sync file between host and VM
+func validateFileSync(ctx context.Context, t *testing.T, profile string) {
+	// 1. create dummy folder for MINIKUBE_HOME
+	oldMinikubeHome := os.Getenv("MINIKUBE_HOME")
+	defer os.Setenv("MINIKUBE_HOME", oldMinikubeHome)
+
+	home := os.Getenv("HOME")
+	fmt.Println("HOME is ", home)
+	minikubeHome, err := ioutil.TempDir(home, "")
+	if err != nil {
+		t.Fatalf("error during creating tmp dir: %v", err)
+	}
+	defer os.RemoveAll(minikubeHome)
+	os.Setenv("MINIKUBE_HOME", minikubeHome)
+	fmt.Println("Setting  MINIKUBE_HOME  ", minikubeHome)
+
+	// 2. start VM
+	startArgs := append([]string{"start", "-p", profile, "--wait=true", "--alsologtostderr", "-v=3"}, StartArgs()...)
```

Comment:
I recommend not to start a VM for this !

what you could do is, add the files before other Serial tests 
and then let other Serial tests start minikube and this one in the end should check if the files are in there.

Reply:
I recommend moving the test setup to the serial part
```
https://github.com/kubernetes/minikube/blob/5509edb1d38352c9b0e3101fad523186e93e0550/test/integration/functional_test.go#L62
```

we have two kind of integraiton  test serial and parallel, you could put some files in the folder after other serial tests have start a minikube ( for example before proxy test you can put the files, and then after it starts, check for the files to be there)

this will save tons of time in integraiton tests to not spin up a new VM